### PR TITLE
added careful copy job with corresponding test.

### PR DESCRIPTION
**Pull request by ManInFez**
**Task**
#416 

**Approach**
_Short description of the approach_

**Pre un-WIP checklist**
- [x] Statoil tests pass locally
- [ ] Have completed graphical integration test steps

**Depends on**
* Statoil/libecl#

### CODE REVIEW

**Comment by ghost** [comment]
Can one of the admins verify this patch?

**Comment by joakim-hove** [comment]
Ok to test

File: python/res/fm/shell/shell.py [hunk 1]
```diff
@@ -212,3 +212,9 @@ def copyFile(src , target = None):
                 shutil.copyfile( src , target_file )
         else:
             raise IOError("Input argument:'%s' does not correspond to an existing file" % src)
+
+    @staticmethod
+    def carefulCopyFile(src , target = None):
+        if os.path.exists(target):
+            raise IOError("Input argument:'%s' already exist" % target)
```
Comment (joakim-hove) [inline python/res/fm/shell/shell.py]: That the target file exists is perfectly OK - not exception.

Comment (joakim-hove) [inline python/res/fm/shell/shell.py]: That the target file exists is perfectly OK - not exception.

File: share/ert/forward-models/shell/CAREFUL_COPY_FILE [hunk 2]
```diff
@@ -0,0 +1,9 @@
+STDOUT      careful_copy_file.stdout
+STDERR      careful_copy_file.stderr
```
Comment (joakim-hove) [inline share/ert/forward-models/shell/CAREFUL_COPY_FILE:2]: The std file descriptors are redirected by default - not necessary to do it explicitly.

**Review by joakim-hove — CHANGES_REQUESTED** [review]
Some minor changes - then she goes in. Good

File: python/res/fm/shell/shell.py (hunk 1 above)
Comment (ManInFez) [reply python/res/fm/shell/shell.py]: Should the function fail silently? Would it not be a good idea to let the user know if the copy failed?

Comment (joakim-hove) [reply python/res/fm/shell/shell.py]: Well - if the file is already present the copy should not proceed, and that is the expected operation of this function - i.e. *not* a failure. Could add a message like:
```
print("File: {} already present - not updated".format(target_file))
```

Comment (ManInFez) [reply python/res/fm/shell/shell.py]: Very good, I will change it accordingly.

**Review by joakim-hove — APPROVED** [review]

**Pull request merged**